src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.ts

```diff
@@ -236,4 +240,47 @@ export class QuizExamSubmissionComponent extends ExamSubmissionComponent impleme
         }, this);
         this.studentSubmission.isSynced = false;
     }
+
+    /**
+     * Randomize the order of the questions
+     * (and answerOptions or dragItems within each question)
+     * if randomizeOrder is true
+     *
+     * @param quizExercise {object} the quizExercise to randomize elements in
+     */
+    randomizeOrder(quizExercise: QuizExercise) {
```

Comment:
this code is copied from quiz-participation.component.ts
Can you please move it into some service to avoid duplication?